docker-compose.yml

```diff
@@ -0,0 +1,311 @@
+    container_name: record-streams-uploader
+    user: root
+    restart: unless-stopped
+    depends_on:
+      minio:
+        condition: service_started
+    networks:
+      - mirror-node-bridge
+    environment:
+      DEBUG: "true"
+      REAPER_ENABLE: "true"
+      REAPER_MIN_KEEP: "1"
+      REAPER_INTERVAL: "5"
+      S3_ENABLE: "true"
+      GCS_ENABLE: "false"
+      SIG_REQUIRE: "true"
+      SIG_PRIORITIZE: "true"
+      SIG_EXTENSION: "rcd_sig"
+      STREAM_EXTENSION: "rcd"
+      BUCKET_NAME: "hedera-streams"
+      BUCKET_PATH: "recordstreams/record0.0.3"
+      S3_ACCESS_KEY: "minioadmin"
+      S3_SECRET_KEY: "minioadmin"
+    volumes:
+      - "${NETWORK_NODE_LOGS_ROOT_PATH}/recordStreams/record0.0.3:/records"
+    command:
+      - /usr/bin/env
+      - python3.7
+      - /usr/local/bin/mirror.py
+      - --linux
+      - --watch-directory
+      - /records
+      - --s3-endpoint
+      - http://minio:9000
+      - --debug
+
+  account-balances-uploader:
+    image: gcr.io/hedera-registry/uploader-mirror:0.4.0
+    container_name: account-balances-uploader
+    user: root
+    restart: unless-stopped
+    depends_on:
+      minio:
+        condition: service_started
+    networks:
+      - mirror-node-bridge
+    environment:
+      DEBUG: "true"
+      REAPER_ENABLE: "true"
+      REAPER_MIN_KEEP: "1"
+      REAPER_INTERVAL: "5"
+      S3_ENABLE: "true"
+      GCS_ENABLE: "false"
+      SIG_REQUIRE: "true"
+      SIG_PRIORITIZE: "true"
+      SIG_EXTENSION: "pb_sig"
+      STREAM_EXTENSION: "pb"
+      BUCKET_NAME: "hedera-streams"
+      BUCKET_PATH: "accountBalances/balance0.0.3"
+      S3_ACCESS_KEY: "minioadmin"
+      S3_SECRET_KEY: "minioadmin"
+    volumes:
+      - "${NETWORK_NODE_LOGS_ROOT_PATH}/accountBalances/balance0.0.3:/balances"
+    command:
+      - /usr/bin/env
+      - python3.7
+      - /usr/local/bin/mirror.py
+      - --linux
+      - --watch-directory
+      - /balances
+      - --s3-endpoint
+      - http://minio:9000
+      - --debug
+
+  minio:
+    image: minio/minio
+    container_name: minio
+    command: server /data --console-address ":9001"
+    environment:
+      MINIO_ROOT_USER: "${MINIO_ROOT_USER}"
+      MINIO_ROOT_PASSWORD: "${MINIO_ROOT_PASSWORD}"
+      MINIO_ACCESS_KEY: minioadmin
+      MINIO_SECRET_KEY: minioadmin
+    networks:
+      - mirror-node-bridge
+    ports:
+      - "9000:9000"
+      - "9001:9001"
+    volumes:
+      - "${MINIO_DATA_PATH}:/data"
+
+  db:
+    image: "${MIRROR_POSTGRES_IMAGE}"
+    container_name: mirror-node-db
+    environment:
+      POSTGRES_HOST_AUTH_METHOD: scram-sha-256
+      POSTGRES_INITDB_ARGS: --auth-host=scram-sha-256
+      GRPC_PASSWORD: mirror_grpc_pass
+      IMPORTER_PASSWORD: mirror_importer_pass
+      OWNER_PASSWORD: mirror_node_pass
+      POSTGRES_PASSWORD: postgres_password
+      REST_PASSWORD: mirror_api_pass
+      ROSETTA_PASSWORD: mirror_rosetta_pass
+    networks:
+      - mirror-node-bridge
```

Comment:
I'm not sure your intent for the `mirror-node-bridge` network, but right now it encompasses the uploaders, minio and all mirror node components. I think we should separate that more and have a `mirror-node` network that contains `db`, `importer`, `grpc`, `rest` and `web3`. Then importer only could be on a different network that is common with minio.

Reply:
The two networks are added since `network-node-potgres` and `db` to not interfere with each other as both are on same port. 
@steven-sheehy  Do you mean to add third `mirror-node` network for clarity?

Reply:
@Neeharika-Sompalli Steven is suggesting adding a third network (eg: `uploader-bridge`) that includes minio, the uploaders and the mirror node importer. The mirror node importer would wind up being a member of two networks so it could access the PostgreSQL database. 

Therefore, the importer would have a network section that looks something like this:
```
networks:
   - mirror-node-bridge
   - uploader-bridge
```

Reply:
What Nathan said, although disagree upon the naming:

```yaml
networks:
  - mirror-node
  - cloud-storage
```

Reply:
Fixed. Thanks @steven-sheehy @nathanklick